pages/knowledge/screen-readers/mobile/reading-websites/README.md

```diff
@@ -5,16 +5,18 @@ position: 1
 
 # How to read websites using a mobile screen reader
 
-**For a sighted person, it's hard to imagine how a screen reader user surfs the internet. For a web developer with accessibility in mind, it's crucial to have a basic knowledge about this topic. So be ready to get a concrete idea of the main strategies available to read a website using a mobile screen reader!**
+**For a sighted person, it’s hard to imagine how a screen reader user surfs the internet. For a web developer with accessibility in mind, it’s crucial to have a basic knowledge about this topic. So be ready to get a concrete idea of the main strategies available to read a website using a mobile screen reader!**
 
 [[_TOC_]]
 
 ## Opening a website
 
-We advise you to activate your mobile screen reader only when reading and interacting with a web page. For anything else like entering an URL into the browser's address bar we advise you to deactivate it temporarily; as soon as the web page has loaded, start it again.
+We advise you to activate your mobile screen reader only when reading and interacting with a web page. For anything else like entering an URL into the browser’s address bar we advise you to deactivate it temporarily; as soon as the web page has loaded, start it again.
 
-- For VoiceOver/iOS, use the accessibility shortcut by tripple clicking the Home button (if you haven't done this yet, go back and read [VoiceOver/iOS configuration](/setup/screen-readers/voiceover-ios/)).
-- For TalkBack, on recent Android versions, the shortcut is operated by pressing both volume keys for 3 seconds (if you haven't done this yet, go back and read [TalkBack configuration](/setup/screen-readers/talkback/)).
+- For VoiceOver/iOS, use the accessibility shortcut by triple-clicking the Home button (or the Side button if you’re using an iOS device without a Home button).<br>
```

Comment:
@christophdubach, I'm still on a home-button device, but is it possible that Regina's original suggestion using `power button` would be more helpful as there might still be multiple `side button`s?

Reply:
Apple calls it the side button. 
And since you have to press one of the volume buttons at the same time to turn the phone off, power button wouldn't be entirely accurate. So I'd rather stick with side button.

![image](https://github.com/user-attachments/assets/f29fa939-6572-4caa-b717-aba20d276605)

Reply:
Oh, thanks, makes perfect sense.